src/__swaps__/utils/swaps.ts

```diff
@@ -46,8 +46,10 @@ import { IS_APK_BUILD } from 'react-native-dotenv';
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 import isTestFlight from '@/helpers/isTestFlight';
 import { NativeCurrencyKey } from '@/entities';
+import { IS_TEST } from '@/env';
 
 const DEFAULT_SLIPPAGE_BIPS = 500;
+const TEST_MODE_SLIPPAGE_BIPS = 9999; // 99.99% slippage for test mode
```

Comment:
we do this on the bx as well, was seeing a lot of failures on swaps that were caused by low slippage in the testing environment. this seems to help a ton.